pallets/gear/src/tests.rs

```diff
@@ -12424,6 +12424,7 @@ fn module_instantiation_error() {
 }
 
 #[test]
+#[ignore = "issue #3100"]
```

Comment:
we are able to use embedded executor for tests now, aren't we?

Reply:
Yes, we are